3.2/Dockerfile

```diff
@@ -33,26 +30,25 @@ RUN yum install -y centos-release-scl && \
     chmod g+w -R /var/opt/rh/rh-mongodb32/lib/mongodb && \
     chmod -R g+rwx /var/lib/mongodb
 
-# Get prefix path and path to scripts rather than hard-code them in scripts
-ENV CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/mongodb \
+ENV MONGODB_VERSION=3.2 \
+    HOME=/var/lib/mongodb \
+    # Set paths to avoid hard-coding them in scripts.
+    CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/mongodb \
     MONGODB_PREFIX=/opt/rh/rh-mongodb32/root/usr \
-    ENABLED_COLLECTIONS=rh-mongodb32
+    # Incantations to enable Software Collections on `bash` and `sh -i`.
+    ENABLED_COLLECTIONS=rh-mongodb32 \
+    BASH_ENV="\${CONTAINER_SCRIPTS_PATH}/scl_enable" \
+    ENV="\${CONTAINER_SCRIPTS_PATH}/scl_enable" \
+    PROMPT_COMMAND=". \${CONTAINER_SCRIPTS_PATH}/scl_enable"
 
-# When bash is started non-interactively, to run a shell script, for example it
-# looks for this variable and source the content of this file. This will enable
-# the SCL for all scripts without need to do 'scl enable'.
```

Comment:
I didn't "remove" them, just made it less verbose and added a IMPOV important information: it only works for `bash` and `sh -i`. We have hit hard walls forgetting about the `-i` some times...

See https://github.com/sclorg/s2i-python-container/pull/108 for a PoC that we could remove this requirement to wrap everything on a shell.